apps/tlon-mobile/src/screens/ChatListScreen.tsx

```diff
@@ -6,14 +6,15 @@ import {
   ChatList,
   ChatOptionsSheet,
   ContactsProvider,
+  FloatingActionButton,
   GroupPreviewSheet,
   Icon,
   ScreenHeader,
-  Spinner,
   StartDmSheet,
   View,
 } from '@tloncorp/ui';
 import { useCallback, useMemo, useState } from 'react';
+import ContextMenu from 'react-native-context-menu-view';
```

Comment:
This looks kinda weird on android, pops out on the top right:
![Screenshot_1717682814](https://github.com/tloncorp/tlon-apps/assets/1221094/d5db3ced-10b8-47b6-a5e3-2a23f4c9eb93)

Reply:
Moving to new issue.